full-service/src/service/sync.rs

```diff
@@ -284,7 +285,8 @@ pub fn sync_account_next_chunk(
             }
 
             TransactionLog::update_pending_exceeding_tombstone_block_index_to_failed(
-                end_block_index + 1,
+                &account_id,
+                end_block_index,
```

Comment:
Why is this end_block_index here, on the view_only account clause, while below on the regular account (line 379) it is end_block_index + 1?

Reply:
I think + 1 is correct in both places.

Reply:
yeah. the view_only account code isn't covered by the unit test, and if one takes away the + 1 on the code that is covered, the unit test fails.

Reply:
good catch, not sure how I accidentally dropped that
Thanks!